docs/pages/admin-guides/access-controls/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Okta App and Group Sync
+description: Explains how to enable the Okta app and group sync integration, which imports Okta configurations into the Teleport RBAC system.
+---
+
+Okta has its own permissions system that doesn't directly map into Teleport. This
+includes permissions granted by Okta groups along with the assignment of users to individual
+applications within Okta. To model this within Teleport, an administrator would typically need
+to carefully craft a labeling system to attach to various Okta apps and groups and a set of
+roles to go along with them.
+
+With the synchronization of Okta apps and groups as Teleport Access List feature in the Okta
+integration, this work is performed for you. This guide explains how to set up
+Okta app and the sync.
+
+## How it works
+
+In Teleport, the Okta Access List synchronizer looks for any Okta group with
+members or any Okta application with individual assignments that matches filters
+that you can optionally supply during Okta integration enrollment. The
+synchronizer creates the following resources for each matched Okta group or
+application:
+
+- A role for members that grants access to the group/application.
+- A role for owners that grants the ability to review access to the group/application.
+- An Access List representing membership to the group/application.
+- Members for the Access List.
+
+It should be noted that the Access List sync waits until the Okta groups and Okta applications
+has finished syncing as Teleport resources, so it may not start synchronizing immediately on startup.
+
+Only Okta user groups with assignments will be imported as Teleport Access
+Lists. If an Okta User Group has no assignments, it will not be imported until
+it has assignments. If the last user is removed from the Access List, the Access
+List will be removed from Teleport on the next sync.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- Teleport Identity enabled on your account.
+
+- An Okta authentication connector. 
+
+  <Admonition>
+  Before following the guided app and group sync integration flow, you must have
+  completed the following:
+  - [Guided Okta single sign-on flow](./guided-sso.mdx).
+  - [Guided Okta user sync flow](./user-sync.mdx).
+  </Admonition>
+
+- (Optional) The Okta SCIM integration. You can follow the [guided integration
+  flow](./scim-integration.mdx).
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- An Okta organization with admin access.
+
+Enabling the Okta integration will make Teleport take ownership over app and
```

Comment:
Now that Bidirectional Sync is implemented as well, might be worth adding wording for that here ("enabling the integration with bidirectional sync enabled...")